docs/python-sdk.md

```diff
@@ -0,0 +1,238 @@
+
+logging.warning("Logged warning %s %s", 12, True)
+```
+
+The Serverless SDK automatically instruments the `logging.warning` method to
+capture warnings. This makes instrumentation easier as you may already be using
+`logging.warning` to display warnings.
+
+This method only supports capturing strings.
+
+We recommend avoiding using unique instance values for the strings. For example,
+if you need to include a userId, email, request ID, or any ID that may be unique
+to the individual invocation, we recommend using Tagging instead.
+
+This method will capture the stack trace of the `logging.warning` call so it
+is easy to identify in Console.
+
+### Tagging
+
+#### Setting Tags on the Trace
+
+```python
+serverless_sdk.set_tag("userId", user_id)
+```
+
+Using the `set_tag` method will create Tags associated with the entire Trace.
+You'll be able to see the Tags on the Trace Details page in the Trace Explorer.
+
+All Tags set with `set_tag` are also inherited by all the Captured Errors and
+Captured Warnings. 
+
+Tag keys may only contain alphanumeric, `.`, `-`, and `_` characters. Tag values
+may contain any string value. Invalid tag keys will not throw errors, instead,
+an SDK error will be made available in Dev Mode and Trace Details.
+
+#### Settings Tags with console.error and console.warn
+
+```python
+import logging
+
+serverless_sdk.set_tag("userId", user_id)
+
+logging.error("Logged error")
+logging.warning("Logged warning %s %s", 12, True)
+```
+
+Using `set_tag` sets the Tag values on both the Trace and all Captured Errors
+and Captured Warnings. Captured Errors and Captured Warnings can be created
+using the `logging.error` and `logging.warning` methods. Therefore, Tags set
+with `set_tag` will apply to all Captured Errors and Captured Warnings
+created using `logging.error` and `logging.warning`.
+
+#### Setting Tags on Captured Errors
+
+```python
+serverless_sdk.capture_error(
+    Exception("Captured error"),
+    tags={"userId": "example", "invocationId": invocation_id},
+)
+```
+
+Tags can also be set on the individual error. If you previously set a Tag using
+`set_tag` then the Tags set on `capture_error` will override the Tags on the
+Captured Error, while keeping the Tag on the trace unmodified.
+
+Tag keys on `capture_error` are validated the same way as tag keys on
+`set_tag`.
+
+#### Setting Tags on Captured Warnings
+
+```python
+serverless_sdk.capture_warning(
+    "Captured warning",
+    tags={"userId": "example", "invocationid": invocation_id},
+)
+```
+
+Tags can also be added on the individual Captured Warnings, just like Captured
+Errors.
+
+Tag keys on `capture_warning` are validated the same way as tag keys on
+`set_tag`.
+
+### Capturing Unhandled Exceptions with Flask
+
+Serverless Console will capture unhandled exceptions thrown from the handler
+method. This can be achieved without including the `serverless_sdk` package, as
+it this is provided by the AWS Lambda Layer added to your Lambda function when
```

Comment:
Grammar fix: "as it this is provided" => "as it is provided"

Reply:
Fixed